### ISSUE TITLE
Textbelt as a user-facing public service

### ISSUE THREAD

**Issue by typpo** [typpo/textbelt#49]
Create a form that allows users to send texts directly from the website.

**Comment by daluu** [comment]
FYI, for adding a CAPTCHA, we can use this perhaps: https://www.google.com/recaptcha

**Comment by daluu** [comment]
FYI, if this is just about offering UI/GUI front ends to Textbelt, I already made some a while back, as browser extensions, etc. You can point folks to them:

https://github.com/daluu/textbelt-clients/wiki/Textbelt-clients%2C-installation-and-download

note that this probably enforces rate limiting at the user/client side (user's own public IP per their ISP).

**Comment by typpo** [comment]
Agreed this would be great. Hoping someone makes a pull request for this :)

**Comment by daluu** [comment]
It looks like @esmith-ssd did some work but didn't make a formal pull request? Or did it incorrectly.

As for my UI clients, I had those as a separate project, think it better to have separate than be a core part of Textbelt. But users of Textbelt API who are using self hosting option are welcome to customize those clients for a self hosted version as I have them pointed to the public service by default.

**Comment by esmith-ssd** [comment]
@daluu It was a failed attempt at a pull request to the original project. It is an AJAX front end application (very basic), that makes a POST call to the web service and returns the response. I just confirmed that it is still working. It could be quickly altered to take advantage of #47 (adding  carrier). However, it may not be viable with  the CAPTCHA, since it is all on the client side. I removed the css links but verified, at the time of submission, that the css from the main textbelt website would make it look "pretty".
